conta/models.py

```diff
@@ -4,7 +4,7 @@
 
 
 class ContaCorrente(TimeStampedModel):
-    id = models.UUIDField(default=uuid.uuid4, unique=True, primary_key=True)
+    id_conta = models.UUIDField(default=uuid.uuid4, unique=True, primary_key=True)
```

Comment:
Por que mudar o nome do campo?

Reply:
Estava causando conflito quando ia criar variáveis. Assim o campo fica com o mesmo nome das variáveis atribuidas.

Reply:
Nao eh recomendado alterar nome de campos de model pois os mesmos geram migrations e quebra de contrato em outros campos. Ao definir variaveis, vc pode alterar o nome das variveis em ci, porem a da model deve ser sempre a mesma. 

Por exemplo: 
Caso tu queira receber o id da conta e salvar numa variavel pra buscar na tabela, voce pode criar uma variavel chamada id_conta e guardar la, porem quando for dar um get, por exemplo, na model vc usaria ContaCorrente.objects.get(id=account_id). Isso nao vai causar nenhum conflito com outras variaveis e tambem nao vai gerar conflito com palavras reservadas.